config/CMakeLists.txt

```diff
@@ -16,10 +18,30 @@ foreach (CONFIG__BASE__XLEN IN ITEMS 32 64)
             install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${config_filename}
                 DESTINATION ${CMAKE_INSTALL_DATADIR}/${CMAKE_PROJECT_NAME}/config
             )
+
+            # After the emulator is built, run it to verify this config is valid.
+            add_custom_command(
```

Comment:
Why not set it with `POST_BUILD` 

```cmake
add_custom_command(sail_riscv_sim POST_BUILD
 COMMAND something
)
```

Reply:
Because I didn't know about it! :-D Seems like a good idea. This makes it a bit trickier:

> The <target> must be defined in the current directory; targets defined in other directories may not be specified.

but I'll try it.